jira/client.py

```diff
@@ -2140,6 +2141,19 @@ def project_permissionscheme(self, project: str) -> PermissionScheme:
         """
         return self._find_for_resource(PermissionScheme, project)
 
+    @translate_resource_args
+    def project_priorityscheme(self, project: str) -> PriorityScheme:
+        """Get a PriorityScheme Resource from the server.
+
+
+        Args:
+            project (str): ID or key of the project to get the PriorityScheme for
+
+        Returns:
+            PriorityScheme,: The priority scheme
```

Comment:
```suggestion
            PriorityScheme: The priority scheme
```

Reply:
Sure. I'm struggling to get credentials for the CI Jira Cloud account though. I wasn't sure if I should create yet another pair of accounts (I see 2 pair already) or try to recover/reset credentials for the cloud CI. For local testing I'll use my own account.

Reply:
Yep I can send those to you on element, but if your individual account works that is fine too.
It's a bit annoying at the moment as only branches created from this project run the Cloud CI (limitation of Github sharing the credentials it seems) but if the server tests pass that is good enough to start with.